content/_config.yml

```
@@ -0,0 +1,13 @@
+title: "Operate-first tutorials"
```

Let's keep the repo name here:

```suggestion
title: Hitchhiker's Guide to Operate First
```

---

Resolved in latest commit